### ISSUE TITLE
Update to support Shopware v6.5.1.x?

### ISSUE THREAD

**Issue by woerki** [Shape-and-Shift/shopware-esd#80]
A lot of updates incl security fixes had been released, but this plugin does still support Shopware v6.4.x
Is any update planned?!

**Comment by jan040m** [comment]
would be interesting to hear if you are still working on the plugin or not because it is not listed in the shopware store at the moment

**Comment by ChristopherDosin** [comment]
Yes - We're working on the update and currently testing it. There's already an open PR for the 6.5 compability:
https://github.com/Shape-and-Shift/shopware-esd/pull/82

**Comment by jan040m** [comment]
ah great to hear! I assume that the plugin will be available in the shopware store again when you managed the shopware 6.5. compatibility?

**Comment by ChristopherDosin** [comment]
That's correct. Shopware disabled the plugin, because we didn't had a 6.5 compability on time.

**Comment by jan040m** [comment]
thanks for your quick reply and your information
